src/Socket_p.h

```diff
@@ -548,7 +545,7 @@ namespace Arcus
 
         google::protobuf::io::ArrayInputStream array(wire_message->data, wire_message->size);
         google::protobuf::io::CodedInputStream stream(&array);
-        stream.SetTotalBytesLimit(message_size_maximum, message_size_warning);
+        stream.SetTotalBytesLimit(message_size_maximum);
```

Comment:
In Protobuf 3.0, this function still has the second parameter. The overload you're using was added in version 3.5, so we can't use it.